examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart

```diff
@@ -59,6 +59,17 @@ class PuppiesAppBar extends StatelessWidget implements PreferredSizeWidget {
             .add(PuppyListFilterEvent(query: query)),
       );
 
+  Widget _createTitle(BuildContext context) {
+    final lastQuery = context.read<PuppyListBloc>().lastSearchedQuery;
+    return lastQuery != ''
```

Comment:
@StanislavPrime, check out this method: [isNotEmpty](https://api.dart.dev/stable/2.3.2/dart-core/String/isNotEmpty.html). By using it, you would improve the readability

Reply:
I changed `lastQuery != ''`  to `lastQuert.isNotEmpty`